service/service.py

```diff
@@ -69,14 +69,20 @@
     "related-product-id",
     type=int,
     required=False,
-    help="List Recommendations by related product id",
+    help="List Recommendations by related product id"
 )
 recommendation_args.add_argument(
     "type-id", type=int, required=False, help="List Recommendations by type id"
 )
 recommendation_args.add_argument(
     "status", type=inputs.boolean, required=False, help="List Recommendations by status"
 )
+recommendation_args.add_argument(
```

Comment:
Also if we delete relationship endpoint we can also delete these two arguments right?

Reply:
Actually I added these two args mostly for the unitests... Are we going to delete the endpoint of the relationship?

Reply:
You should rename the product1 and product2 in the unit tests to use "product-id" and "related-product-id" you can delete the 80-85 lines, no need for these